templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} Manufacturer</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input class="btn btn-primary" type="submit" value="Submit">
+    <button type="button" class="btn btn-secondary" onclick="window.history.back();">Cancel</button>
```

Comment:
Great job including a 'Cancel' button that leads to the previous page. This meets the checklist requirement.